Makefile

```diff
@@ -527,7 +527,7 @@ docker-compose-build-swagger: awx/projects docker-compose-sources
 SCHEMA_DIFF_BASE_BRANCH ?= devel
 detect-schema-change: genschema
 	curl https://s3.amazonaws.com/awx-public-ci-files/$(SCHEMA_DIFF_BASE_BRANCH)/schema.json -o reference-schema.json
-	# Ignore differences in whitespace with -b
+  # Ignore differences in whitespace with -b
```

Comment:
[nit] is this change intentional

Reply:
Sort of. Can you tell if those are spaces or tabs in the other one? I have a habit of using spaces for "make" stuff and only using tabs for "command" blocks because anything that begins with tabs will be passed to the shell. Maybe it's just more spaces than my editor converted it to, or maybe it's a tab that I converted to spaces. I'll check.

Reply:
Went ahead and just reversed that hunk